src/harness/testcases/WINNF_FT_S_SCS_testcase.py

```diff
@@ -181,6 +183,38 @@ def test_WINNF_FT_S_SCS_10(self,config_filename):
       # Check Registration Response
       self.assertEqual(response[0]['response']['responseCode'], 104)
 
+  def generate_SCS_11_default_config(self, filename):
+    """Generates the WinnForum configuration for SCS_11. """
+    # Create the configuration for blacklisted client cert/key,wait timer information
+
+    config = {
+      'clientCert': self.getCertFilename("blacklisted_client.cert"),
+      'clientKey': self.getCertFilename("blacklisted_client.key"),
+      'waitTimer': 60
```

Comment:
Why is the wait timer needed? I thought we would either use a new CRL file (in which case the SAS needs to pull the CRL file before establishing the TLS connection) or add the cert to the CRL manually (in which case we count on the tester to manually wait 7 days before executing this test case).

Reply:
The   timer is not needed now. It has been removed. The SAS UUT needs to pull from   the simple CRL server and update its CRL before this test is run.